content/operate/guides/cosmos/syncing.md

```diff
@@ -4,6 +4,281 @@ title: Syncing
+```
+
+`firehose-cosmos` provides scripts for setting up and running the following mainnet networks,
+however these are limited in the range of supported Operating systems and configuration so we recommend following the guides for getting these chains running.
+
+
+These scripts are only really used for testing and should not be used for running this process.
+
+- [Cosmoshub4](https://github.com/figment-networks/firehose-cosmos/blob/main/devel/cosmoshub4)
+- [Osmosis1](https://github.com/figment-networks/firehose-cosmos/blob/main/devel/osmosis1)
+
+---
+
+## Install and run instrumented nodes
+
+In order to index Cosmos based nodes with `firehose-cosmos`, you will need to insure you are using the correctly modified binaries for the chain you are targetting. Figment provide these for CosmosHub and you can find them on the `Gaia-DM` releases page located [here](https://github.com/figment-networks/gaia-dm/releases). You will need to also include the following in your node configuration file:
+
+```YAML
+#######################################################
+###       Extractor Configuration Options           ###
+#######################################################
+[extractor]
+enabled = true
+output_file = "stdout"
+```
+
+## Firehose-Cosmos Configuration
+
+If you wish to use a configuration file instead of setting all CLI flags, you may create a new firehose.yml file in your current working directory.
+You can point to your instrumented binary using the `ingestor-node-path` flag and this will start the binary for you.
+
+Example:
+
+```YAML
+start:
+  args:
+    - ingestor
+    - merger
+    - relayer
+    - firehose
+  flags:
+    # Common flags
+    common-first-streamable-block: 1
+
+    # Ingestor specific flags
+    ingestor-mode: node
+    ingestor-node-path: path/to/gaiad_or_osmosisd
+    ingestor-node-args: start --x-crisis-skip-assert-invariants
+    ingestor-node-env: "KEY=VALUE,KEY=VALUE"
+```
+
+---
+
+### Logs input mode
+
+It's possible to run the firehose ingestor from the static logs, mostly for development/testing purposes.
+
+Example config:
+
+```YAML
+start:
+  args:
+    - ingestor
+    - merger
+    - relayer
+    - firehose
+  flags:
+    # ... other config options
+
+    # Ingestor specific flags
+    ingestor-mode: logs
+    ingestor-logs-dir: /path/to/logs/dir
+
+    # Configure the pattern if not using .log extension
+    # ingestor-logs-pattern: *.log
+```
+
+---
+
+## Binary configuration
+
+As the Cosmos based binaries have been modified, you will need to include the following in your node configuration file:
+
+```YAML
+#######################################################
+###       Extractor Configuration Options           ###
+#######################################################
+[extractor]
+enabled = true
+output_file = "stdout"
+```
+
+## Syncing your chain
+
+One you begin running `firehose-cosmos`, you will begin to see your blocks streaming through if it is successfully instrumented.
```

Comment:
Add code block for start command ? Should we also talk about firehose.yml or it's already covered elsewhere in the doc? If so reference to it.